jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -155,26 +154,22 @@ public ImageAndAuthorization call()
         } catch (RegistryUnauthorizedException registryUnauthorizedException) {
           // The registry requires us to authenticate using the Docker Token Authentication.
           // See https://docs.docker.com/registry/spec/auth/token
-          try {
-            Optional<RegistryAuthenticator> registryAuthenticator =
-                buildConfiguration
-                    .newBaseImageRegistryClientFactory()
-                    .newRegistryClient()
-                    .getRegistryAuthenticator();
-            if (registryAuthenticator.isPresent()) {
-              Authorization pullAuthorization =
-                  registryAuthenticator.get().authenticatePull(registryCredential);
-
-              return new ImageAndAuthorization(
-                  pullBaseImage(pullAuthorization, progressEventDispatcher), pullAuthorization);
-            }
-
-          } catch (InsecureRegistryException insecureRegistryException) {
-            // Cannot skip certificate validation or use HTTP; fall through.
+          Optional<RegistryAuthenticator> registryAuthenticator =
+              buildConfiguration
+                  .newBaseImageRegistryClientFactory()
+                  .newRegistryClient()
+                  .getRegistryAuthenticator();
+          if (registryAuthenticator.isPresent()) {
+            Authorization pullAuthorization =
+                registryAuthenticator.get().authenticatePull(registryCredential);
+
+            return new ImageAndAuthorization(
+                pullBaseImage(pullAuthorization, progressEventDispatcher), pullAuthorization);
           }
           eventHandlers.dispatch(
               LogEvent.error(
-                  "Failed to retrieve authentication challenge for registry that required token authentication"));
+                  "Failed to retrieve authentication challenge for registry that required token "
```

Comment:
should this line not have just fit? like before?

Reply:
Yeah, this line goes over 100 columns. Sad.